### ISSUE TITLE
ELI5 explanation

### ISSUE THREAD

**Issue by gnubyte** [llSourcell/Learn_Deep_Learning_in_6_Weeks#2]
The problems I encounter while watching these videos is that there is enough tabs open that I have to spend an hour researching an idea of an idea of an idea.

Can we begin to assemble some "explain it to me like Im five" explanations here? I think it would help streamline the process. We can use this issue to crowdsource the explanations and collaborate on it.

**Comment by gnubyte** [comment]
So for example, chapter 1 opens to back propagation. Here are questions that I think we should begin answering in a separate file in the repo in markdown: 
 - [ ] What is back propagation?
 - [ ] Why is it relevant?
 - [ ] Where would I use back propagation?
 - [ ] Where can I apply this in my 9/5, typical daily setting, for an average IT Helpdesk, System administrator, Web dev, or programmer? (Notably outside the realm of data analyst)

_this distinction of target audience allows us to work outside the vacuum of assumptions that 'x' person has a background, and allows us to make readers digests/summaries, for those who are short on time or have life responsibilities such as kids that demand more time of them_

**Comment by Lovejonezin** [comment]
Well I think the course makes the assumption that if one has an interest in the subject matter, they will follow their own questions to get the answer best suited for their needs. 

Speaking of myself, I had a base knowledge in deep learning from the other Siraj Raval YT videos, so the supplemental information with this course is   sufficient for me. Back propagation is one of the  largest parts of the neural network, more than likely you've heard it before even beginning to code one.

I do wish the first exercise didn't skip around so much. Just explain the lines of code as it's being entered. I wasted a week out of frustration and decided to come back later to figure it out.
